var/spack/repos/builtin/packages/molgw/package.py

```diff
@@ -103,8 +104,6 @@ def edit(self, spec, prefix):
             flags["FC"] = self.compiler.fc
 
         # Set FCFLAGS
-        if self.compiler.flags.get("fflags") is not None:
-            flags["FCFLAGS"] = " ".join(self.compiler.flags.get("fflags")) + " "
```

Comment:
Why were these flags removed?

Reply:
Hi @alecbcs

I removed the two lines because I got the following error:
```
==> Error: AttributeError: 'CompilerAdaptor' object has no attribute 'flags'
The 'molgw' package cannot find an attribute while trying to build from sources. This might be due to a change in Spack's package format to support multiple build-systems for a single package. You can fix this by updating the build recipe, and you can also report the issue as a bug. More information at https://spack.readthedocs.io/en/latest/packaging_guide.html#installation-procedure

```

Spack changed something and `self.compiler.flags` is not working anymore.

What is the correct and up-to-date way to get the Fortran compiler default flags?
Thanks.
Fabien

Reply:
@alalazo what is the best way to get the Fortran flags post compilers as nodes? Did we miss this package in our migration?

Reply:
This is the only package using them directly, so possibly an overlook in:

https://github.com/spack/spack/blob/72b14de89e6c38c45396aecfe57edba68f7306b3/lib/spack/spack/compilers/adaptor.py#L20

where we have to add this property. I'll add the property, if it is needed, but I'm wondering if this package can't use the compiler wrapper as the vast majority of other packages, and use:

```python
def flag_handler(self, name, flags):    
```

to inject more flags. Would that be possible?

Reply:
Thank you your replies.

I want to have the package as standard as possible. There is no reason not to use the compiler wrapper.
I just do not know how to do it in a clean way.

Can you point to me a Fortran package that is showing the canonical way of doing? And I'll copy the strategy.
Thanks.

Reply:
My PR removes a python line that was not working any more.
The access to fortran flags through `self.compiler.flags.get("fflags")` is not possible anymore.
These flags are actually not needed by the package.